packages/affiliates/libs/affiliates.js

```diff
@@ -144,32 +147,49 @@ const DeployerRewards = ({ queries, empCreatorAbi, empAbi, coingecko, synthPrice
     }, []);
 
     // Per snapshot calculate the associated amount that each deployer is entitled to.
-    let finalDeployerPayouts = valuePerSnapshot.reduce((result, valueByEmp) => {
-      const totalValueLocked = valueByEmp.reduce((result, [, value]) => {
-        return result.add(toBN(value));
-      }, toBN("0"));
-      valueByEmp.forEach(([emp, value]) => {
-        const deployer = empDeployers.get(emp);
-        const contribution =
-          totalValueLocked.toString() != "0"
-            ? toBN(value) // eslint-disable-line indent
-                .mul(toBN(toWei("1"))) // eslint-disable-line indent
-                .div(totalValueLocked) // eslint-disable-line indent
-            : toBN("0"); // eslint-disable-line indent
-        const rewards = contribution.mul(payoutPerSnapshot).div(toBN(toWei("1")));
-        if (result[deployer] == null) result[deployer] = toBN("0");
-        result[deployer] = result[deployer].add(rewards);
-      });
+    let finalPayouts = valuePerSnapshot.reduce(
+      ({ deployerPayouts, empPayouts }, valueByEmp) => {
+        const totalValueLocked = valueByEmp.reduce((result, [, value]) => {
+          return result.add(toBN(value));
+        }, toBN("0"));
+        valueByEmp.forEach(([emp, value]) => {
+          const deployer = empDeployers.get(emp);
+          const contribution =
+            totalValueLocked.toString() != "0"
+              ? toBN(value) // eslint-disable-line indent
+                  .mul(toBN(toWei("1"))) // eslint-disable-line indent
+                  .div(totalValueLocked) // eslint-disable-line indent
+              : toBN("0"); // eslint-disable-line indent
+          const rewards = contribution.mul(payoutPerSnapshot).div(toBN(toWei("1")));
+
+          // calculate deployer rewards
+          if (deployerPayouts[deployer] == null) deployerPayouts[deployer] = toBN("0");
```

Comment:
I notice you check if objects are explicitly equal to `null`, why not just shorten the if statements to `if(!object)` instead? Also you should use javascript's triple equal signs

Reply:
Yea that `if(!object)` would be equivalent, I can consider changing to that, I just do it this way out of habit. 
I use `==` rather than `===` in this case because I want to check for null or undefined (falsey values). Not the best because it will also catch 0s. I  agree in general though `===` is preferred.